packages/apps/plugins/plugin-client/src/ClientPlugin.tsx

```diff
@@ -81,7 +81,8 @@ export const ClientPlugin = ({
   // TODO(burdon): Document.
   registerSignalFactory();
 
-  const settings = new LocalStorageStore<ClientSettingsProps>('dxos.org/settings');
+  const settings = new LocalStorageStore<ClientSettingsProps>('dxos.org/settings', { automerge: true });
```

Comment:
Does this now make Automerge the default backend? I know this was discussed on Friday during a call, but probably worth calling out. Maybe in PR description?